get_all_manifests.sh

```diff
@@ -6,20 +6,20 @@ GITHUB_URL="https://github.com"
 # component: notebook, dsp, kserve, dashbaord, cf/ray/kueue/trainingoperator, trustyai, modelmesh, modelregistry.
 # in the format of "repo-org:repo-name:ref-name:source-folder:target-folder".
 declare -A COMPONENT_MANIFESTS=(
-    ["codeflare"]="opendatahub-io:codeflare-operator:main:config:codeflare"
-    ["ray"]="opendatahub-io:kuberay:dev:ray-operator/config:ray"
-    ["kueue"]="opendatahub-io:kueue:dev:config:kueue"
-    ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:main:config:data-science-pipelines-operator"
-    ["odh-dashboard"]="opendatahub-io:odh-dashboard:main:manifests:dashboard"
-    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
-    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
-    ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
-    ["trustyai"]="trustyai-explainability:trustyai-service-operator:main:config:trustyai-service-operator"
+    ["codeflare"]="opendatahub-io:codeflare-operator:v1.7.0-odh:config:codeflare"
+    ["ray"]="opendatahub-io:kuberay:v1.1.1-08-14-2024:ray-operator/config:ray"
+    ["kueue"]="opendatahub-io:kueue:v0.7.0-odh-1:config:kueue"
+    ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:v2.5.x:config:data-science-pipelines-operator"
```

Comment:
@AjayJagan so after https://issues.redhat.com/browse/RHOAIENG-11687, we are still using branch but not tag? 
same for kserve and model-controller?

Reply:
I think these are dependent on what the components provide

Reply:
so that is kind of related. But for the automation to start using tags or branches(flexible) we need to get this in : https://github.com/opendatahub-io/opendatahub-operator/pull/1199